cypress/utils/dashboards/commands.js

```diff
@@ -0,0 +1,106 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+Cypress.Commands.add('selectFromDataSourceSelector', (dataSourceTitle, dataSourceId) => {
+  // clean up the input field
+  cy.wait(1000);
```

Comment:
:( cy.wait make tests flaky

Reply:
we don't use this one. it is for easy migration.